### PR TITLE
Block for nvidia gpu.

### PR DESCRIPTION
**Pull request by lgAshanl**
For proprietary nvidia drivers. Has no dependencies on xorg.

### CODE REVIEW

**Comment by greshake** [comment]
Very nice! Thanks :+1:

**Comment by atheriel** [comment]
This looks great!

File: blocks.md [hunk 1]
```diff
@@ -338,6 +338,40 @@ Key | Values | Required | Default
...
+```
+
+### Options
+
+Key | Values | Required | Default
+----|--------|----------|--------
+`gpu_id` | GPU id in system | No | 0
+`label` | Display custom gpu label | No | ""
+`interval` | Update interval, in seconds. | No | `1`
+`utilization` | Display gpu utilization. In percents. | No | `true`
```
Comment (atheriel) [inline blocks.md]: Should these be `show_utilization`, `show_memory`, `show_temperature`, `show_fan_speed`, and `show_clocks` instead? That would fit the style of other blocks more closely.

Comment (lgAshanl) [reply blocks.md]: I have not seen this in other blocks, but this is logical.

Comment (atheriel) [reply blocks.md]: I should clarify: the style we *aspire* to with other blocks. Thanks for the changes!

**Pull request merged**